src/main/java/mekanism/common/content/teleporter/TeleporterValidator.java

```diff
@@ -0,0 +1,56 @@
+package mekanism.common.content.teleporter;
+
+import it.unimi.dsi.fastutil.longs.Long2ObjectMap;
+import mekanism.common.content.blocktype.BlockType;
+import mekanism.common.lib.math.voxel.VoxelCuboid;
+import mekanism.common.lib.multiblock.CuboidStructureValidator;
+import mekanism.common.lib.multiblock.FormationProtocol;
+import mekanism.common.lib.multiblock.FormationProtocol.CasingType;
+import mekanism.common.lib.multiblock.StructureHelper;
+import mekanism.common.registries.MekanismBlockTypes;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.chunk.IChunk;
+
+public class TeleporterValidator extends CuboidStructureValidator<TeleporterMultiblockData> {
+
+    public static final VoxelCuboid MIN_BOUNDS = new VoxelCuboid(1, 1, 1);
+    public static final VoxelCuboid MAX_BOUNDS = new VoxelCuboid(17, 18, 17);
```

Comment:
I don't think these bounds are correct at all.

Reply:
Yeah. Those were just copied from elsewhere and adjusted while I tried to get it to work.

Reply:
Honestly I think a new validator that isn't directly a CuboidValidator may be better/more effective as realistically we want a VoxelPlane not a VoxelCuboid and there isn't a great way to specify the min bounds for a flat thing using the cuboid system.

Reply:
Yeah. I found the structure concept a bit hard to follow so I was trying to get to a working state again before I explored something more fit to purpose.